pkg/test/kube/dump.go

```diff
@@ -129,17 +130,35 @@ func DumpPodEvents(a Accessor, workDir, namespace string, pods ...kubeApiCore.Po
 func DumpPodLogs(a Accessor, workDir, namespace string, pods ...kubeApiCore.Pod) {
 	pods = podsOrFetch(a, pods, namespace)
 
+	var sb strings.Builder
 	for _, pod := range pods {
+		isVM := checkIfVM(pod)
 		containers := append(pod.Spec.Containers, pod.Spec.InitContainers...)
 		for _, container := range containers {
 			l, err := a.Logs(pod.Namespace, pod.Name, container.Name, false /* previousLog */)
 			if err != nil {
 				scopes.Framework.Errorf("Unable to get logs for pod/container: %s/%s/%s", pod.Namespace, pod.Name, container.Name)
 				continue
 			}
+			sb.WriteString(l)
+
+			// Get proxy logs if the pod is a VM, since kubectl logs only shows the logs from iptables
+			if isVM && container.Name == "istio-proxy" {
+				if cfgDump, err := a.Exec(pod.Namespace, pod.Name, container.Name, "pilot-agent request GET config_dump"); err == nil {
```

Comment:
we already capture this somewhere else, what we are looking for is the log written to file. So we can probably do:

```
cat /var/log/istio/istio.err.log
```
and 
```
cat /var/log/istio/istio.log
```
as part of the log

Reply:
Oh so we actually want the logs from the pod? Understood. Will update!